core/core-auto-completion.el

```diff
@@ -11,12 +11,14 @@
 
 ;; Company -------------------------------------------------------------------
 
+(setq company-backends '((company-dabbrev-code company-gtags company-etags company-keywords)
```

Comment:
Better use `defvar`, because then Emacs will show the original value if user changed the variable.